moduleroot/.github/workflows/pr_test.yml.erb

```diff
@@ -92,42 +106,22 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Activate Ruby 2.7
-      uses: actions/setup-ruby@v1
+      uses: ruby/setup-ruby@v1
```

Comment:
Good catch :)

Reply:
Since the meat of the two files is the same, I took on the habit of fixing one, and then copying over everything to the other (except the preamble and the slack notification) :shrug: 

can't wait for the day when actions support higher-level re-use - I don't want to re-compose all of this into a singular action just for this...

Reply:
> can't wait for the day when actions support higher-level re-use - I don't want to re-compose all of this into a singular action just for this...

Oh yes, absolutely. Templating and copying over to many repos is painful.